templates/concert-ticket/templates/main.tpl

```diff
@@ -0,0 +1,168 @@
+                <text-chunk font-size="9">Aerosmith, Iron Maiden, Avenged Sevenfold, Kiss, Motley Crue, Fleetwood Mac, Tom Petty, Kid Rock, Shinedown,</text-chunk>
+                <text-chunk font-size="9">Five Finger Death Punch, Rob Zombie, Korn &amp; many more rock legends have graced this permanent Main Stage over the course of the last two and a half decades.</text-chunk>
+            </paragraph>
+        </division>
+    </table-cell>
+    <table-cell>
+        <image src="path('./templates/res/map.png')" height="160" width="100" margin="5 0 0 0"></image>
+    </table-cell>
+    <table-cell>
+        <division>
+            <paragraph>
+                    <text-chunk font="helvetica-bold" font-size="12">How to find us</text-chunk>
+            </paragraph>
+            <paragraph margin="5 0 0 0" line-height="1.3">
+                <text-chunk font-size="9">Concert Hall is located three blocks west of the park and three blocks north the library.</text-chunk>
+                <text-chunk font-size="9">Free parking is available nearby, which fills up close to showtime, and on the streets around.</text-chunk>
+                <text-chunk font-size="9">Bike racks are located outside the main entrance tothe Hall.</text-chunk>
+            </paragraph>
+            <paragraph margin="10 0 0 0">
+                <text-chunk font="helvetica-bold" font-size="12">Learn More At</text-chunk>
+            </paragraph>
+            <paragraph margin="5 0 0 0">
+                <text-chunk font-size="9">Facebook /ConcertHall</text-chunk>
+            </paragraph>
+            <paragraph margin="2 0 0 0">
+                <text-chunk font-size="9">Twitter @ConcertHall</text-chunk>
+            </paragraph>
+            <paragraph margin="2 0 0 0">
+                <text-chunk font-size="9">Instagram @ConcertHall</text-chunk>
+            </paragraph>
+        </division>
+    </table-cell>
+</table>
+
+<table columns="2">
+    <table-cell vertical-align="middle">
+        <division margin="0 0 0 10">
+            <paragraph>
+                <text-chunk font-size="9">This is your ticket. Print this Entire page, fold it and bring it with you to the event.</text-chunk>
+                <text-chunk font-size="9"> Please make sure the QR code is visible.</text-chunk>
+            </paragraph>
+            <image src="path('./templates/res/ticket-img.png')" height="80" width="100" margin="5 0 0 0"></image>
+        </division>
+    </table-cell>
+    <table-cell>
+        <division margin="10 0 0 0">
+            {{template "header" dict "FontSize" 19 "SubFontSize" 12}}
+            <table columns="2">
+            <table-cell>
+                    <division margin="5 0 0 0">
+                        <table columns="2">
+                            {{range .Detail}}
+                            {{if ne .FieldName "address"}}
```

Comment:
If you change the `address` field name to `Address`, probably need to update here as well.